README.md

```diff
@@ -21,7 +21,7 @@ Sphinx supplies a number of useful
 ["directives"](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html)
 (The sphinx equivalent of LaTeX commands) for stuff like code highlighting,
 remarks, warnings and so on.
-
+export PATH="/Users/ivan/Library/Python/3.9/bin"
```

Comment:
Something is not right with the PR. I can see release note on the screenshot you made but if I look at files changed, I don't see which files you have changed and what is the content. Also, this path seems to point to something locally?

Reply:
My bad, wrong commit. Fixed